src/components/Text.js

```diff
@@ -95,6 +107,11 @@ export const RegularText = styled(Text)`
 
 export const BoldText = styled(RegularText)`
   font-family: titillium-web-bold;
+  ${(props) =>
+    props.italic &&
+    css`
+      font-family: titillium-web-bold-italic;
+    `};
```

Comment:
as `BoldText` is based on `RegularText` `italic` is already working, since it's part of regular.
can be removed here.

Reply:
it is still needed because the italic prop of regular text uses the regular font weight and not the bold font weight

Reply:
have you tried out? you can do `<BoldText italic ...` because of the dependence

Reply:
ok. i tried it out now and i actually overlooked a thing.

yes the `italic` prop gets passed down.

no it does not do anything.

the font-family is already overwritten by the font-family set `BoldText` which the `italic` prop for `BoldText` then overwrites again.

so removing this results in the text becoming normal bold text which is no longer italic, even if we set the `italic` prop.

Reply:
thanks for detailed explanation.
bold and italic are both set per font-family which i overlooked!
so `titillium-web-bold-italic` is introduced. i have not read before, that it is "bold-italic". i thought of a combination of the bold style and an italic style.

thanks. no everything makes sense!

Reply:
:+1: